static/app/components/forms/fields/secretField.tsx

```diff
@@ -3,7 +3,12 @@ import InputField, {InputFieldProps} from './inputField';
 export interface SecretFieldProps extends Omit<InputFieldProps, 'type'> {}
 
 function SecretField(props: SecretFieldProps) {
-  return <InputField {...props} type="password" />;
+  // XXX: We explicitly give the password field a aria textbox role. This field
+  // does not typically have a role, but for testability reasons it's preferred
+  // for it to have a role. See [0]
```

Comment:
why an array? 😄 I like it

Reply:
haha, this is just something I picked up awhile ago from some other codebase comments. no idea where tbh.